iothub/device/src/Transport/AmqpIoT/AmqpIoTErrorAdapter.cs

```diff
@@ -205,7 +197,7 @@ public static Exception ToIotHubClientContract(Error error)
             string message = error.Description;
 
             string trackingId = null;
-            if (error.Info != null && error.Info.TryGetValue(TrackingId, out trackingId))
+            if (error.Info != null && error.Info.TryGetValue(AmqpIoTConstants.TrackingId, out trackingId))
             {
```

Comment:
would "AmqpIoTConstants.TrackingId" always be a valid value?

Reply:
It is a const `com.microsoft:tracking-id`.